src/adapters/suggest.js

```diff
@@ -50,14 +48,16 @@ export default class Suggest {
                 this.bragiPromise, this.historyPromise, this.categoryPromise
               ])
 
+              if (!bragiResponse)
+                return resolve(null)
+
+              this.suggestList = []
+
               if (categoryResponse)
                 this.suggestList = this.suggestList.concat(categoryResponse)
 
-              if (bragiResponse) {
-                this.bragiPromise = null
-                this.suggestList = this.suggestList.concat(bragiResponse)
-              }
-
+              this.bragiPromise = null
+              this.suggestList = this.suggestList.concat(bragiResponse)
               this.suggestList = this.suggestList.concat(storeResponse)
```

Comment:
it looks good to me.

Not directly related to this change: "favorite" POIs from store are rendered twice in the suggestions, because a `else if` is missing in the rendering function (line 82)

Reply:
oh yes, a missmerge i think, thanks !